1699: Card design koblenz

f1sh1918 opened the pull request:
### Short description

Adjust the koblenz card design

### Proposed changes

<!-- Describe this PR in more detail. -->

- replace logos
- set theme color
- adjust card branding
- add extra upscale for header logo

Note: I adjusted the header right logo a bit because fine adjustments with the current card branding are difficult. At some point we gonna have to create separate card template widgets per project

<img src="https://github.com/user-attachments/assets/8795ca7d-45ea-47b2-ad9a-8b80567ecee8" width=35% height=20%>

<img src="https://github.com/user-attachments/assets/a3500426-25f5-4e73-ab6f-f5047155fc6e" width=35% height=20%>


### Side effects

<!-- List all related components that have not been explicitly changed but may be affected by this PR -->

- none

### Testing

1. Start koblenz local and activate card
2. Check the other apps if the card design doesn't break

### Resolved issues

<!-- List all issues which should be closed when this PR is merged. -->

Fixes: #1699 
Fixes: #1741

steffenkleinle commented on frontend/build-configs/koblenz/index.ts:
```diff
@@ -18,8 +18,8 @@ export const koblenzCommon: CommonBuildConfigType = {
     theme: {
-        primaryLight: "#8377A9",
-        primaryDark: "#8377A9",
+        primaryLight: "#d10074",
```
:thought_balloon: I want a theme color :D Not related to your PR though

steffenkleinle reviewed (approved):
Tested on android, looks good

seluianova reviewed (commented):
Is a pink background intended here? (I know we get it from the theme, but anyway)
![image](https://github.com/user-attachments/assets/5212b6c3-bcb7-49ec-a02d-43fc0097571a)

f1sh1918 commented:
> Is a pink background intended here? ![image](https://private-user-images.githubusercontent.com/115008338/380819960-5212b6c3-bcb7-49ec-a02d-43fc0097571a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxOTMxMzUsIm5iZiI6MTczMDE5MjgzNSwicGF0aCI6Ii8xMTUwMDgzMzgvMzgwODE5OTYwLTUyMTJiNmMzLWJjYjctNDllYy1hMDJkLTQzZmMwMDk3NTcxYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOVQwOTA3MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMGQyNDhmM2E5MDI1ZWRmM2RjODZhODQyNDg3NWM2ZTU2MTllYWUwNjJmYTlmODhiNDA2NmE3ZjNlNWQwZTE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.yJSCJJ81_LBsI3DCBIu_nzWGA24mL0WM2DsQ_i9Tc3w)

The issue here is that material3 themes use surfaceTintColor. Usually a very light version of the primary color is used.
Since for koblenz the color is darker its still visible here.
The issue ist that if i override this color in general on lots of positions it will not be used like
<img width="306" alt="image" src="https://github.com/user-attachments/assets/b8abf5f6-8ee3-42e8-99a4-6d874e26eba4">

So probably i have to override this color just for the dialogs

seluianova commented:
> I will talk to toni in the afternoon

ready for review or you still want to clarify/update anything?

seluianova commented:
just found another issue:
the card is not rendered correctly on the card verification.
![image](https://github.com/user-attachments/assets/bfaae617-2ace-44de-8f4b-deb8e0661b4c)

```
======== Exception caught by rendering library =====================================================
The following assertion was thrown during layout:
A RenderFlex overflowed by 0.767 pixels on the bottom.

The relevant error-causing widget was: 
  Column Column:file:///home/tory/IdeaProjects/entitlementcard/frontend/lib/identification/id_card/card_content.dart:181:32
```

ztefanie reviewed (commented):
Did just a quick review and did not test as two people already did. lgtm. Would have liked to have a background (image) on the card, but just personal preference.

f1sh1918 commented:
> Did just a quick review and did not test as two people already did. lgtm. Would have liked to have a background (image) on the card, but just personal preference.

Yes same for me but we don't have a proper design for that. This also leads to the issue that page background and card are both white and don't have a good contrast :(
Maybe we can improve that at a later point

f1sh1918 commented:
@seluianova i added a tint color to override this slight pink color for dialogs.
Fixed also the pixel overflow.
Additional design optimization like colors and contrasts will be done here
https://github.com/digitalfabrik/entitlementcard/issues/1742
